content/03.text-mining.md

```diff
@@ -21,34 +21,67 @@
-   4. 1706.01556v2
-   5. ^^ A few papers here but a lot more will be put into place 
-   6. Mention caveat which is the need for large annotated datasets
-   7. Mention a direction the field is moving to which is weak supervision and more that info that will come in time.
+#### Supervised Relationship Extraction
+
+Supervised extraction uses labeled data to learn patterns and map example input to desired output.
+Most of these approaches have flourished due to pre-labelled publicly available datasets (Table {@tbl:supervised-text-datasets}).
+These datasets were constructed by curators for shared open tasks [@raw:biocreative/chemprot; @doi:10.1093/database/baw068] or as a means to provide the scientific community with a gold standard [@doi:10.1093/bioinformatics/btl616; @doi:10.1093/database/baw068; @doi:10.1186/1471-2105-14-323].
+Approaches that use these available datasets range from using support vector machines (SVMs) with custom kernels to deep learning with algorithms that can construct their own features.
+In the rest of this section we discuss approaches that use supervised methods to detect relationship-asserting sentences.
+
+Extracting relationships in a supervised setting can involve mapping textual input onto a high dimensional space.
+Support vector machines are a type of classifier that can accomplish this task by utilizing a mapping function called a kernel [@doi:10.1186/s13326-017-0168-3; @doi:10.1371/journal.pcbi.1004630].
+These kernels take information such as a sentence's dependency tree [@doi:10.1093/database/bay108; @doi:10.1093/bioinformatics/btw503], part of speech tags [@doi:10.1186/s13326-017-0168-3] or even word counts [@doi:10.1371/journal.pcbi.1004630] and directly maps them onto a dense feature space.
+Within this space a hyperplane can be constructed to separate sentences into the positive class (mentions a relationship) or negative class (does not mention a relationship). 
+Kernels can be manually constructed or selected to cater to the relationship being extracted [@doi:10.1186/s13326-017-0168-3; @doi:10.1093/bioinformatics/btw503;@doi:10.1371/journal.pcbi.1004630; @doi:10.1371/journal.pcbi.1004630].
+Determining the correct kernel requires expert knowledge to be successful and is a nontrivial task depending on the relationship.
+Besides single kernel methods, a recent study used an ensemble of SVMs to extract disease-gene associations [@doi:10.1371/journal.pone.0200699].
+This ensemble outperformed notable disease-gene association extractors [@doi:10.1016/j.jbi.2015.08.008; @doi:10.1186/s12859-015-0472-9] in terms of precision, recall and even F1 score.
+Overall, future SVM methods should consider a more automated approaches to define these kernel functions or develop ways to use semi-automatic methods for kernel construction.
+
+Deep learning is an increasingly popular class of techniques that can construct their own features within a high dimensional space [@raw:GoodfellowDL; @doi:10.1038/nature14539].
```

Comment:
This would be a good transition in if you modify the final sentence of the previous paragraph to note that being able to do nonlinear feature combination automatically is valuable.